### PR TITLE
ACMS-935: ACMS site install using existing configuration is not working

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**

Fixes #[ACMS-935](https://backlog.acquia.com/browse/ACMS-935)

**Proposed changes**

- Ignore required configuration which are causing issue.
- Allow config ignore to ignore configuration on export.

**Alternatives considered**

N/A

**Testing steps**

Follow [ACMS install](https://docs.google.com/document/d/1rgMn2VhLw18__HskVjJ3Cbx9Ia-trIyTLzcAM9CksQ8/edit#heading=h.idh8r9wxfxy2)

**Merge requirements**
- [ _Bub_] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -161,6 +161,9 @@
             "drupal/acquia_telemetry-acquia_telemetry": {
                 "Add check for core index": "https://www.drupal.org/files/issues/2020-08-18/3165473-27.patch"
             },
+            "drupal/config_ignore": {
```
Comment (vishalkhode1) [inline composer.json:164]: @chandan-singh7929 Please update composer.json file for acquia_cms_site_studio module as well and include this patch there as well.
@kdruckman Do we also need to pin the config_ignore module in composer.json as we've added the patch ?

File: config/optional/config_ignore.settings.yml [hunk 2]
```diff
@@ -2,3 +2,7 @@ ignored_config_entities:
   - 'language.entity.*'
   - 'media.type.image:uuid'
   - 'crop.type.focal_point:uuid'
+  - 'cohesion.settings'
+  - 'purge.plugins'
+  - 'purge.logger_channels'
```
Comment (vishalkhode1) [inline config/optional/config_ignore.settings.yml:7]: @chandan-singh7929 Should we also add a update hook to update config ignore configs for the user who are already using Acquia CMS (or Custom profile) ?
cc @kdruckman

Comment (chandan-singh7929) [reply config/optional/config_ignore.settings.yml:7]: Done

File: acquia_cms.profile [hunk 3]
```diff
@@ -287,3 +288,15 @@ function install_acms_set_favicon() {
...
+/**
+ * Update config ignore settings.
+ */
+function acquia_cms_update_8001() {
+  $module_path = \Drupal::moduleHandler()->getModule('acquia_cms')->getPath();
+  $config_optional = $module_path . '/config/optional';
+  $source_optional_dir = new FileStorage($config_optional);
+
+  $config_storage = \Drupal::service('config.storage');
+  $config_storage->write('config_ignore.settings', $source_optional_dir->read('config_ignore.settings'));
```
Comment (vishalkhode1) [inline acquia_cms.profile:301]: @chandan-singh7929 I think we can’t re-import the complete config_ignore config, as others might have added/removed some configs. Those changes will be then vanished. We've add the config changes in config_ignore config (if those are not already present).

**Pull request merged**